### PR TITLE
Installed Tailwind

### PR DESCRIPTION
**Pull request by karencardoz**
Installed Tailwind
Designed Home Page using CSS Styling

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -38,101 +36,14 @@ function Button({
   shadow, // boolean
   ...rest
 }) {
-
```
Comment (spencer-rangle) [inline src/components/Button/Button.js:41]: Why was all this removed?  Could add on your own Button component, or perhaps rename this to StyledButton, something like that, and keep the button you made.
See comment below for your implementation of the button.

File: src/components/Button/Button.js [hunk 2]
```diff
@@ -38,101 +36,14 @@ function Button({
...
-    return buttonStyling;
-
-  }
-
-  // ButtonTypesEnum
-
-
   return (
-    <button {...rest} className={chooseButtonClasses()}>
+    <button
```
Comment (spencer-rangle) [inline src/components/Button/Button.js]: This styling is good for this particular instance of a button, but is not generalizable enough for a Button component.  If you don't intend to reuse this, then could put this styling right on the homepage.

**Review by spencer-rangle — CHANGES_REQUESTED** [review]

File: src/components/Button/Button.js (hunk 1 above)
Comment (karencardoz) [reply src/components/Button/Button.js:41]: oops reverted to the previous Button.js

**Review by spencer-rangle — APPROVED** [review]

File: src/components/Button/Button.js (hunk 2 above)
Comment (karencardoz) [reply src/components/Button/Button.js]: will tackle i a new PR

**Pull request merged**